### PR TITLE
Upm 6658/cli1 5 0

### PR DESCRIPTION
**Pull request by zozowell**
## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [x] This PR is for a release, please add this tag: v1.5.0

### CODE REVIEW

**Review by drothery-edb — APPROVED** [review]
@zozowell I made some copy edits. If you agree with them, I can merge this and get it into today's build.

File: product_docs/docs/biganimal/release/reference/cli.mdx [hunk 1]
```diff
@@ -184,32 +184,65 @@ You can use the `--clusterConfigFile` command to create one or more clusters wit
...
+  ```
+  allowIpRangeMap: [["9.9.9.9/28", "Allow traffic from App A"],["10.10.10.10/27", "Allow traffic from App B"]]
+  pgConfigMap: [["application_name","test_app"],["array_nulls","true"]] 
+  ```
+!!!
+
 To create the cluster using the sample configuration file `config_file.yaml`:
 
-```shell
+```yaml
```
Comment (zozowell) [inline product_docs/docs/biganimal/release/reference/cli.mdx:222]: @drothery-edb , why does this become `yaml`? i think it a simple `shell`

Comment (drothery-edb) [inline product_docs/docs/biganimal/release/reference/cli.mdx:222]:
```suggestion
```shell
```

**Comment by zozowell** [comment]
Looked good to me.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**